src/js/components/Search.jsx

```diff
@@ -173,11 +174,35 @@ class Search extends Component {
         </div>
       )
     }
+    
+    if (searchResults.error) {
+      const {message} = searchResults.error
+      const spanStyle = {...style.span, color: 'red'}
+      let text = {}
+       
+      try {
```

Comment:
Another point here is why are we interpretting the "data" sent back from the rest side. why doesn't the rest side just figure out what to send and send it back as a string?